cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -143,9 +143,8 @@ jobs:
           echo "EOF" >> "$GITHUB_OUTPUT"
       - name: "Linkage report"
         run: |
-          cargo dist linkage ${{ matrix.dist_args }} --print-output --print-json > linkage.json
-
-          echo "linkage=$(cat linkage.json)" >> "${GITHUB_OUTPUT}"
+          jq .linkage dist-manifest.json > linkage.json
+          cargo dist linkage ${{ matrix.dist_args }} --from-json=linkage.json --print-output
```

Comment:
oh interesting. at this point do we just want `cargo dist build` to have a flag to include linkage info in the human readable output?

Reply:
Yeah, that's definitely an option! So long as it doesn't get buried in the rest of the output, I guess.